core/src/wsv.rs

```diff
@@ -157,6 +159,68 @@ impl WorldStateView {
         tokens
     }
 
+    /// Returns a set of permission tokens granted to this account separately.
```

Comment:
```suggestion
    /// Return a set of permission tokens granted to this account separately.
```

by `separately` you mean not part of any role or what?

Reply:
yes

Reply:
```
/// Return a set of permission tokens granted to this account individually.
```
Maybe?

Reply:
I'd suggest mentioning the role explicitly 

```suggestion
    /// Return a set of permission tokens granted to this account not as part of any role.
```